static/app/components/codeSnippet.tsx

```diff
@@ -1,26 +1,154 @@
-import 'prism-sentry/index.css';
-
-import {Fragment, useEffect, useRef, useState} from 'react';
+import {useEffect, useRef, useState} from 'react';
+import {css} from '@emotion/react';
 import styled from '@emotion/styled';
 import Prism from 'prismjs';
 
-import Tooltip from 'sentry/components/tooltip';
+import Button from 'sentry/components/button';
 import {IconCopy} from 'sentry/icons';
 import {t} from 'sentry/locale';
 import space from 'sentry/styles/space';
+import {Theme} from 'sentry/utils/theme';
+
+/**
+ * Prism styles with support for light/dark mode, to be used as an alternative
+ * to 'prism-sentry' and 'prism.css'.
+ */
+export const prismStyles = ({theme}: {theme: Theme}) => css`
```

Comment:
Exporting because I plan to reuse these styles in Issue Activities to fix this problem:
<img width="798" alt="Screen Shot 2022-09-30 at 12 09 23 PM" src="https://user-images.githubusercontent.com/44172267/193340203-ad935796-05d9-4696-8862-1873c9828767.png">

Reply:
https://github.com/getsentry/sentry/pull/39528